FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -19,6 +22,60 @@ public final class RemoteFeedLoader: FeedLoader {
 	}
 
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
-		fatalError("Must be implemented")
+		client.get(from: url) { [weak self] result in
+			guard let _ = self else {
+				return
+			}
+
+			switch result {
+			case .success((let data, let response)):
+				completion(FeedItemMapper.map(data, from: response))
+			case .failure:
+				completion(.failure(Error.connectivity))
+			}
+		}
+	}
+}
+
+final class FeedItemMapper {
```

Comment:
This class is a `RemoteFeedLoader`'s implementation detail. No need for it to be accessible from other places than this file. 

Please make `FeedItemMapper` `private`.

```suggestion
private final class FeedItemMapper {
```

Reply:
You're right. Updated.